Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -75,7 +75,19 @@ param (
             ThrowError -errorMessage (Get-LocalizedString -Key "Parameter '{0}' cannot be null or empty." -ArgumentList "credential")
         }   
     }
-    
+
+    function Get-FullUserName(
```

Comment:
Can you name this function Get-DownLevelLogonName... Refer: https://msdn.microsoft.com/en-us/library/windows/desktop/aa380525(v=vs.85).aspx.